lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java

```diff
@@ -65,8 +67,17 @@ public static MarkupContent getMarkupDescription(Parameter parameter) {
 			mojosToConsiderList = mojosToConsiderList.stream().filter(mojo -> interestingMojos.contains(mojo.getGoal()))
 					.collect(Collectors.toList());
 		}
+		// Wrap these into MojoParameters that could be used? Might be duplicating work if they're present in the MojoParameters that get retrieved afterwards
 		List<Parameter> parameters = mojosToConsiderList.stream().flatMap(mojo -> mojo.getParameters().stream())
 				.collect(Collectors.toList());
+		MavenProject project =  cache.getLastSuccessfulMavenProject(request.getXMLDocument());
+		// TODO: project could be null..
+		mavenSession.setProjects(Collections.singletonList(project));
+		// TODO: This isin't currently being used, just here for debugging purposes
+		List<MojoParameter> mojoParams = mojosToConsiderList.stream().flatMap(mojo -> {
```

Comment:
you can most likely omit ` { return ` and `}` there as it's a single value expression.